local/devel/style-cleanup.php

```diff
@@ -9,22 +9,30 @@
  *    - set line ends to LF(-only)
  *    - remove ?> and blank lines at end of file
  *    - add missing LF to end of file
- *    - detect characters before <? at start of file
+ *
+ *  It also warns on the following issues:
+ *
+ *    - characters before open tag at start of file
+ *    - short open tags
  *
  *  This script may be run any time to check and clean up the current OC code.
  *
  *  Unicode Reminder メモ
  ***************************************************************************/
 
-$exclude = [
+# The following code is made to run also on the developer host, which may
+# have a restricted environent like an old Windows PHP. Keep it simple and
+# do not include other OC code.
+
+$exclude = array(
     'htdocs/cache',
     'htdocs/cache2',
     'htdocs/lib2/smarty',
     'htdocs/okapi',
     'htdocs/templates2/mail',
     'htdocs/var',
-    'htdocs/vendor'
-];
+    'htdocs/vendor',
```

Comment:
It simplifies maintaining the list, especially when sorting entries. Nice habit from OKAPI coding style. ;)